### PR TITLE
Add System.LoaderOptimizationAttribute

### PR DESCRIPTION
**Pull request by rahku**

### CODE REVIEW

**Comment by rahku** [comment]
PTAL @kouvel @jkotas @danmosemsft

File: src/System.Runtime/src/System/LoaderOptimizationAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    [AttributeUsage (AttributeTargets.Method)]
```
Comment (justinvp) [inline src/System.Runtime/src/System/LoaderOptimizationAttribute.cs]: Nit: Space between `AttributeUsage` and `(` could be deleted.

File: src/System.Runtime/src/System/LoaderOptimizationAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    public sealed partial class LoaderOptimizationAttribute : System.Attribute
```
Comment (justinvp) [inline src/System.Runtime/src/System/LoaderOptimizationAttribute.cs]: Couple nits:
 - Does it need to be `partial` here?
 - `System.Attribute` => `Attribute`

File: src/System.Runtime/src/System/LoaderOptimizationAttribute.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        private byte _val;
+        public LoaderOptimizationAttribute(byte value) 
+        { 
+            _val = value;
+        }
+        public LoaderOptimizationAttribute(System.LoaderOptimization value)
```
Comment (justinvp) [inline src/System.Runtime/src/System/LoaderOptimizationAttribute.cs]: Nit: `System.LoaderOptimization` => `LoaderOptimization`

**Comment by rahku** [comment]
moved. @jkotas PTAL

**Comment by jkotas** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test  please

**Pull request merged**